### PR TITLE
Improving pt-br locale

### PR DESCRIPTION
**Pull request by reebr**
Hello ✋ 

Just adding the some properties to pt-br locale:
- [x] `weekdaysMin`
- [x] `weekStart`

### CODE REVIEW

File: src/locale/pt-br.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import dayjs from 'dayjs'
 const locale = {
   name: 'pt-br',
   weekdays: 'Domingo_Segunda-feira_Terça-feira_Quarta-feira_Quinta-feira_Sexta-feira_Sábado'.split('_'),
+  weekdaysShort: 'Dom._Seg._Ter._Qua._Qui._Sex._Sáb.'.split('_'),
```
Comment (iamkun) [inline src/locale/pt-br.js]: thanks, how about removing all the ending dot to keep the same with 
https://github.com/moment/moment/blob/develop/locale/pt-br.js

Comment (reebr) [reply src/locale/pt-br.js]: Normally we use "." at the end of this weekdays.
But no problem, I'll fix it.

**Comment by reebr** [comment]
@iamkun all this properties from [moment](https://github.com/moment/moment/blob/develop/locale/pt-br.js) works on dayjs? If yes, can I add all that properties to update our portuguese locale?

**Comment by iamkun** [comment]
yes you can see all the supported properties in our locale doc.

**Comment by reebr** [comment]
@iamkun nice.
I've added `weekStart` and `monthsShort` too.

File: src/locale/pt-br.js [hunk 2]
```diff
@@ -3,7 +3,11 @@ import dayjs from 'dayjs'
 const locale = {
   name: 'pt-br',
   weekdays: 'Domingo_Segunda-feira_Terça-feira_Quarta-feira_Quinta-feira_Sexta-feira_Sábado'.split('_'),
+  weekdaysShort: 'Dom_Seg_Ter_Qua_Qui_Sex_Sáb'.split('_'),
+  weekdaysMin : 'Do_2ª_3ª_4ª_5ª_6ª_Sá'.split('_'),
+  weekStart: 1,
   months: 'Janeiro_Fevereiro_Março_Abril_Maio_Junho_Julho_Agosto_Setembro_Outubro_Novembro_Dezembro'.split('_'),
+  monthsShort: 'Jan_Fev_Mar_Abr_Mai_Jun_Jul_Ago_Set_Out_Nov_Dez'.split('_'),
```
Comment (iamkun) [inline src/locale/pt-br.js]: if monthsShort always be the first 3 charts, we could just leave this property blank.

Comment (reebr) [reply src/locale/pt-br.js]: @iamkun sure, done.

**Comment by reebr** [comment]
Also removed `weekdaysShort`, the documentation says it use the first three letters.

**Pull request merged**